packages/shared/lib/core/profile-manager/api/events/newOutput.ts

```diff
@@ -1,3 +1,24 @@
-import { OutputData } from '@iota/wallet'
+import { syncBalance } from '@core/account/actions/syncBalance'
+import { activeAccounts } from '@core/profile/stores'
+import { Activity } from '@core/wallet/classes/activity.class'
+import { addActivityToAccountActivitiesInAllAccountActivities } from '@core/wallet/stores/all-account-activities.store'
+import { Bech32Helper } from '@lib/bech32Helper'
+import { Converter } from '@lib/converter'
+import { get } from 'svelte/store'
+import { NewOutputEvent } from '../types/newOutputEvent'
 
-export function handleNewOutputEvent(payload: OutputData): void {}
+export function handleNewOutputEvent(event: NewOutputEvent): void {
+    for (const account of get(activeAccounts)) {
+        const address =
+            event.output.address?.type === 0
```

Comment:
We should add typings for the address types etc, no idea what address type = 0 is.

Reply:
how would you suggest making it more readable?

Reply:
We could add the types as an enum or get this from the dependency?